doc/src/_static/custom.css

```diff
@@ -0,0 +1,116 @@
+/* Completely disable blockquotes. The Furo defaults make block quotes very*/
+/* apparent. However, the parameter blocks from numpydoc render as block
+ * quotes, which looks bad. Furthermore, it is very easy for things to
+ * accidentally have block quotes due to the fact that RST makes indented
+ * blocks render as a block quote (unlike Markdown which makes indented blocks
+ * render as code). There aren't any places in the documentation that use a
+ * block quote on purpose. */
+blockquote {
+    border-left: inherit;
+    font-style: inherit;
+    margin-left: inherit;
+    margin-right: inherit;
+    padding: inherit;
+    background-color: inherit;
+    margin-block-start: -0.5rem;
+    margin-block-end: -0.5rem;
+}
+
+/* Version selector. See _templates/sidebar/versions.html */
+.versions-header {
+    color: white;
+    padding: var(--sidebar-item-spacing-vertical) var
+             (--sidebar-item-spacing-horizontal);
+    margin: 1rem 0rem;
+}
+.current-version {
+    background-color: #2f431e;
+}
+.versions-not-found {
+    color: white;
+    font-style: italic;
+    padding: var(--sidebar-item-spacing-vertical) var(--sidebar-item-spacing-horizontal);
+}
+
+/* Make the "SymPy documentation" text bold */
+.sidebar-brand-text {
+    font-weight: bold;
+}
+/* Remove the green underline from the "SymPy documentation" on hover */
+.sidebar-brand:hover {
+    text-decoration: none !important;
+}
+
+/* Make top-level items in the sidebar bold */
+.sidebar-tree .toctree-l1>.reference, .sidebar-tree .toctree-l1>label .icon {
+    font-weight: bold !important;
+}
+/* Indicate the current page using a background color rather than bold text */
+.sidebar-tree .current-page>.reference {
+    font-weight: normal;
+    background-color: #2f431e;
+}
+/* Make sure the 'a' colors below don't affect the sidebar*/
+.sidebar-tree .icon, .sidebar-tree .reference {
+    color: var(--color-sidebar-link-text) !important;
+}
+
+/* The "hide search matches" text after doing a search. Defaults to the same
+   color as the icon which is illegible on the green background. */
+.highlight-link a {
+    color: white !important;
+}
+
+/* Make the "class", "property" and so on before function names in the API*/
+/* docs bold. */
+em.property:first-child {
+    font-weight: bold;
+}
```

Comment:
The entire signature is bold by default, so you shouldn't need this.

Reply:
Looks like you are right. This must have been left over from a previous iteration.